File: Java/14. Longest Common Prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        Arrays.sort(strs);
        String str1 = strs[0];
        String str2 = strs[strs.length-1];
        String res="";
        for(int i=0;i<str1.length();i++){
            if(str1.charAt(i) == str2.charAt(i)){
                res+=str1.charAt(i);
            } 
            else {
                break;
            }
        }
        return res;
    }
}

File: JavaScript/14. Longest Common Prefix.js
/**
 * @param {string[]} strs
 * @return {string}
 */
var longestCommonPrefix = function(strs) {
    strs.sort();
    var n=strs.length;
    var str1=strs[0];
    var str2=strs[n-1];
    var res="";
    for(var i=0;i<str1.length;i++){
        if(str1[i]==str2[i]){
            res+=str1[i];
        }
        else{
            break;
        }
    }
    return res;
};
